server/src/interfaces/media.interface.ts

```diff
@@ -126,7 +127,7 @@ export interface ProbeOptions {
 
 export interface IMediaRepository {
   // image
-  extract(input: string, output: string): Promise<boolean>;
+  extract(asset: AssetEntity, outputPath: string): Promise<boolean>;
```

Comment:
Can you change this to take a second argument? 

```ts
extract(output: string, options: { orientation?: number })
```

This interface should not be using AssetEntity.